devfiles/projector-idea-c.yaml

```diff
@@ -1,7 +1,7 @@
 metadata:
-  name: che-ideaic
+  name: projector-idea-community
 components:
   - type: cheEditor
-    reference: 'https://raw.githubusercontent.com/che-incubator/che-editor-intellij-community/master/devfiles/meta.yaml'
-    alias: ideaic-editor
+    reference: 'https://raw.githubusercontent.com/che-incubator/jetbrains-editor-images/meta/che-projector-idea-c.meta.yaml'
```

Comment:
I think we could keep the name 'community' in the filename
about the alias name I think it might be better to keep community as well and remove projector if the length is too long (because from user point of view, you're more interested on what is executed than if it is with vnc or projector)

Reply:
Updated